filament/docs/Versioning.md

```diff
@@ -0,0 +1,15 @@
+## Versioning
+
+Filament uses a 3-number versioning scheme that superficially resembles a [semantic
+version](https://semver.org/) but is actually more interesting because of our material system. Here
+are the guidelines:
+
+- Increment the **most significant** number only when making a non-backwards compatible API change.
```

Comment:
What about new major APIs? 1.0.0 -> 2.0.0.?

Reply:
Okay, I changed the guidelines to allow for this.